### DISCUSSION TITLE
FINN tutorial @ FPL'22

### DISCUSSION THREAD

**Discussion by auphelia** [Xilinx/finn#672] [Announcements]
At this year's FPL conference, we are happy to offer a FINN tutorial (https://fpl.org/tutorialfinn/). 
Besides a general introduction to the FINN project and talks from the FINN community (featuring BMW, VLV Medical, IMT Atlantique and Paderborn University), we also have a hands-on session, in which you can train a small MLP for network intrusion detection with Brevitas and build an FPGA accelerator with the FINN compiler. 

It will be a hybrid event, so I look forward to meeting some of you in person!
You can register via the conference website: https://fpl.org/registration/

**Comment by julienmas** [comment]
Will the tutorial recording be available online afterwards?

**Reply by auphelia** [reply]
Hi @julienmas,

Thanks for your question! 
If you are registered at the FPL, you can access the recordings for a few months after the conference. Whether the recordings will be available outside the conference platform is not yet clear. 

If you are interested in FINN tutorials we have run in general, here are some links, including a recording of the tutorial from FPGA'21: https://xilinx.github.io/finn//2021/01/27/finn-tutorial-fpga21.html
